specs/core/0_fork-choice.md

```diff
@@ -248,10 +254,17 @@ def on_attestation(store: Store, attestation: Attestation) -> None:
     # Cannot calculate the current shuffling if have not seen the target
     assert target.root in store.blocks
 
+    # Attestations target be for a known block. If target block is unknown, delay consideration until the block is found
+    assert target.root in store.blocks
```

Comment:
This line duplicates the check above.

Reply:
Solving in https://github.com/ethereum/eth2.0-specs/pull/1492